template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.0/content/EmptyWeb-CSharp/Program.cs

```diff
@@ -14,11 +15,8 @@ public static void Main(string[] args)
             var host = new WebHostBuilder()
                 .UseKestrel()
                 .UseContentRoot(Directory.GetCurrentDirectory())
-                .UseIISIntegration()
```

Comment:
I don't think we can remove this yet, not until we are confident that the new ANCM has been shipped everywhere. @shirhatti